src/components/MyAccount/Settings/AccountSettingsDisplayOptions.tsx

```diff
@@ -29,7 +30,50 @@ export const AccountSettingsDisplay = ({ patron }: { patron: Patron }) => {
   return <>{terms.map(buildListElementsWithIcons)}</>
 }
 
-export const AccountSettingsForm = ({ patron }: { patron: Patron }) => {
+export const AccountSettingsForm = ({
+  patron,
+  setIsFormValid,
+}: {
+  patron: Patron
+  setIsFormValid: Dispatch<React.SetStateAction<boolean>>
+}) => {
+  const [formData, setFormData] = useState({
+    primaryPhone: patron.phones[0]?.number,
+    email: patron.emails[0],
+  })
+
+  const isFormValid = useCallback(() => {
+    const phoneRegex = /^\d+$/
```

Comment:
This regex would exclude a number formatted like 123-456-7890. I think we should allow people to submit in that format, because it is easier to see when you're typing it out.

Reply:
Allow them to submit in that format but then store without the dashes ?

Reply:
@charmingduchess most recent fix allows dashes, but kind of opens up a whole can of worms with validation (for example, now someone can just submit a bunch of dashes, which will save as the empty string. We would need to require a certain number of numbers in the input to block that) so lmk what you think